frontend/src/metabase/querying/components/FilterContent/FilterBody.tsx

```diff
@@ -0,0 +1,46 @@
+import type * as Lib from "metabase-lib";
+
+import { FilterEmptyState } from "./FilterEmptyState";
+import { TabContent } from "./TabContent";
+import type { GroupItem } from "./types";
+
+type FilterBodyProps = {
+  groupItems: GroupItem[];
+  query: Lib.Query;
+  tab: string | null;
+  version: number;
+  searching: boolean;
+  onChange: (newQuery: Lib.Query) => void;
+  onInput: () => void;
+  onTabChange: (
+    value: ((prevState: string | null) => string | null) | string | null,
+  ) => void;
+};
+
+export const FilterBody = ({
+  groupItems,
+  onChange,
+  onInput,
+  onTabChange,
+  query,
+  searching,
+  tab,
+  version,
+}: FilterBodyProps) => (
+  <>
```

Comment:
nit: you might not need the outer fragment here as both of them only has one element?